File: ui/src/backend/payloads/admin/storepayload.ts
export class StorePayload {
    id: number;
    code: string;
    name: string;
    address: string;
    phone: string;
    email: string;
    active: boolean;

    constructor() {
        this.id = null;
        this.code = null;
        this.name = '';
        this.address = null;
        this.phone = null;
        this.email = null;
        this.active = true;
    }
}

File: api/src/main/java/com/mystore/manager/api/admin/payload/StorePayload.java
package com.mystore.manager.api.admin.payload;

import com.mystore.manager.api.common.payload.GlobalDataPayload;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class StorePayload extends GlobalDataPayload {
    private String address;
    private String phone;
    private String email;
    private boolean active = true;
}
